### DISCUSSION TITLE
Different docker compose for database

### DISCUSSION THREAD

**Discussion by donakhseputa** [JordanKnott/taskcafe#164] [Ideas]
I have docker-compose for the existing database
and I want to taskcafe using my existing database instead from your docker-compose.yml file

how to achieve this??

**Comment by GwynethLlewelyn** [comment]
You cannot. Taskcafé is strictly bound to PostgreSQL and won't work with anything else. See also #167.

**Comment by bfrontend** [comment]
You can modify docker-compose.yml configuration pg's port user and other related information

**Comment by GwynethLlewelyn** [comment]
Oh, I stand corrected — sorry about the confusion (non-native English speaker here!). When I first read this question, years ago, I misunderstood what was meant with an 'existing database'. I wrongly assumed that the OP was asking 'I have my own database management system running on top of X, and I want Taskcafé to connect to it.' That was impossible for anything that wasn't PostgreSQL.

But naturally if the OP _already_ has a PostgreSQL database installed and running, Taskcafé _can_ connect to it easily enough 😁
